lib/datadog/profiling/agent_settings_resolver.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require_relative '../core/configuration/agent_settings_resolver'
+
+module Datadog
+  module Profiling
+    # This class encapsulates any profiling specific agent settings
+    class AgentSettingsResolver < Datadog::Core::Configuration::AgentSettingsResolver
+    end
+  end
+end
```

Comment:
As I mentioned above, I'd like to request that this gets deleted -- I really don't want to maintain my own copy of the `AgentSettingsResolver`.

Reply:
If profiling has no intention of implementing any profiling-specific logic for the agent then I think that's reasonable. I can remove in a follow-up PR.